components/Achievements/Display/index.tsx

```diff
@@ -16,7 +16,7 @@ const centreOffset = boxLength / 2 - strokeWidth / 2
 /**
  * The image display for an achievement including a circular progress bar.
  */
-const Display = ({ unlocked, image, progress, total }: Props): JSX.Element => {
```

Comment:
Wait what's the reason for removing these out of curiosity? I thought specifying the return type would make the code more precise.